docs/parameters.yaml

```diff
@@ -51,40 +44,140 @@ type: bool
-  started by the origin.
-
-  For non-root servers, if $XDG_RUNTIME_DIR is not set, a temporary directory will
-  be created (and removed on shutdown)
+  The name of the file containing a service's private `issuer.jwk` key.
 type: filename
-root_default: /run/pelican/xrootd
-default: $XDG_RUNTIME_DIR/pelican
+root_default: /etc/pelican/issuer.jwk
+default: $ConfigBase/issuer.jwk
+components: ["client", "nsregistry", "director"]
+---
+name: WebAddress
+description: >-
+  A string-encoded IP address that the origin is configured to listen on.
+type: string
+default: 0.0.0.0
 components: ["origin"]
 ---
-name: RobotsTxtFile
+
+############################
+# Federation-Level Configs #
+############################
+name: Federation.DiscoveryUrl
 description: >-
-  Origins may be indexed by web search engines; to control the behavior of search
-  engines, one may provide local policy via a [robots.txt file](https://en.wikipedia.org/wiki/Robots.txt).
+  A URL pointing to the federation's metadata discovery host.
+type: url
+default: none
+components: ["*"]
+---
+name: Federation.TopologyNamespaceUrl
+description: >-
+  A URL containing namespace information for origins and caches configured via the OSG Topology application (a legacy integration). The URL
+  should point to the hosted namespace.json.
+type: url
+osdf_default: https://topology.opensciencegrid.org/stashcache/namespaces.json
+default: none
+components: ["director"]
+---
+name: Federation.DirectorUrl
+description: >-
+  A URL indicating where a director service is hosted.
+type: url
+osdf_default: Default is determined dynamically through metadata at <federation URL>/.well-known/pelican-configuration
+default: none
+components: ["client", "origin"]
+---
+name: Federation.NamespaceUrl
+description: >-
+  A URL indicating where the namespace registry service is hosted.
+type: url
+osdf_default: Default is determined dynamically through metadata at <federation URL>/.well-known/pelican-configuration
+default: none
+components: ["client", "director", "origin"]
+---
 
-  If this file is not present, it will be auto-created with a default policy of
-  blocking all indexing.
-type: filename
-root_default: /etc/pelican/robots.txt
-default: $ConfigBase/robots.txt
+############################
+#   Client-Level Configs   #
+############################
+name: Client.StoppedTransferTimeout
+description: >-
+  A timeout indicating when a "stopped transfer" event should be triggered.
+type: int
+default: 100
+components: ["client"]
+---
+name: Client.SlowTransferRampupTime
+description: >-
+  A duration indicating the rampup period for a slow transfer.
+type: int
+default: 30
+components: ["client"]
+---
+name: Client.SlowTransferWindow
+description: >-
+  A duration indicating the sliding window over which to consider transfer speeds for slow transfers.
+type: int
+default: 30
+components: ["client"]
+---
+name: Client.DisableHttpProxy
```

Comment:
`DisableHttpProxy` and `DisableProxyFallback` are other backward compatibility knobs we should honor.